src/Messaging/NBB.Messaging.Host/MessagingPipeline/ExceptionHandlingMiddleware.cs

```diff
@@ -50,6 +50,7 @@ public async Task Invoke(MessagingContext context, CancellationToken cancellatio
 
                 await _messageBusPublisher.PublishAsync(new
                 {
+                    ExceptionType = ex.GetType(),

```

Comment:
Shouldn't you add ExceptionType for exceptions reported in MessageBusSubscriber

Reply:
done!